interface.py

```diff
@@ -131,6 +154,26 @@ def speedDown(self):
 			self.calcThread.speed -= 1
 			self.controls.updateSpeedDisplay(self.calcThread.speed)
 	
+	def popLimitUp(self):
```

Comment:
I'm not sure that these belong here instead of in grid - something to review later